### DISCUSSION TITLE
Positioning support

### DISCUSSION THREAD

**Discussion by kenjo138** [srsran/srsRAN_Project#164] [General Help]
I am working on a project involving positioning, which is supposed to be supported both on NR revs. 16 and 17. Is there support for this in the current srsRAN build? If so, how would I set this up? Can I access the PRS and SRS data from the log files? Any help would be appreciated.

**Comment by pgawlowicz** [comment]
Hi, currently positioning is not supported. And probably will not be supported in the near future.

**Reply by ljankoschek** [reply]
Hi! I can't find a clear answer to this online for the open source solution. Is there now any positioning support in the current srsRAN build?
